setup_rpi.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# === Load Config ===
+
+# Retrieve the absolute path of the directory containing the script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
please remove this line witch is not used anymore

Reply:
oops.. done !